runtime/Cargo.toml

```diff
@@ -61,7 +61,6 @@ sp-offchain = { workspace = true }
 sp-runtime = { workspace = true }
 sp-session = { workspace = true }
 sp-staking = { workspace = true }
-sp-std = { workspace = true }
```

Comment:
substrate is deprecating sp-std and started to migrate to core and alloc instead.